### DISCUSSION TITLE
UI ehancement

### DISCUSSION THREAD

**Discussion by pierrbt** [VeagleDev/weather-widget#8] [Ideas]
How would we ehance the current UI of the WeathGet application ?
![image](https://user-images.githubusercontent.com/83456236/180102357-d3ecac24-d32b-4e6f-a2a4-639427367e64.png)

**Comment by arthubhub** [comment]
hello, you could write an error message if the connection can not be etablish

**Reply by pierrbt** [reply]
Yes I just have a debug in the console but I surely can easily add an alert window and ask for "retry" or "quit". I'll convert the message into issue for an ehancement

**Comment by arthubhub** [comment]
you could add an icon image

**Reply by pierrbt** [reply]
Yes great idea ! I have already added an icon for the .exe ( for the future v1.8 ) and I can now add an icon for the Window head. You can help me by finding a better icon, and why not some icons for the weather with the clouds, sun, ... (maybe another topic) @arthubhub

**Reply by pierrbt** [reply]
I finished that, you can see it in the new [v1.8 release](https://github.com/mysterious-developers/weather-widget/releases/tag/v1.8)
